Protobuild.Internal/Packages/PackageGlobalTool/KnownToolProvider.cs

```diff
@@ -15,7 +15,7 @@ internal class KnownToolProvider : IKnownToolProvider
         {
             {"jsilc", "http://protobuild.org/hach-que/JSIL"},
             {"swig", "http://protobuild.org/hach-que/SWIG"},
-            {"protobuild.manager", "http://protobuild.org/hach-que/Protobuild.Manager"},
+            {"protobuild.manager", "https-nuget-v3://api.nuget.org/v3/index.json|Protobuild.Manager"},
```

Comment:
Not yet.  SWIG was updated (and there's a separate PR for that that's already been merged), but JSIL is not in NuGet yet.